src/components/common/headers/Header/Header.style.ts

```diff
@@ -0,0 +1,46 @@
+import { Theme } from '@emotion/react';
+import { css } from '@emotion/react';
+import { flexGenerator } from '@styles/generator';
+
+export const headerContainer = css`
+  ${flexGenerator()};
+  width: 100%;
+  padding: 1.95rem 1rem;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+`;
```

Comment:
justify-content, align-items 속성 flexGenerator 이용해서 주시면 좋을 것 같습니다.

```suggestion
export const headerContainer = css`
  ${flexGenerator('row', 'justify-content', 'align-items')};
  width: 100%;
  padding: 1.95rem 1rem;
  gap: 1rem;
`;```